python/src/SubsetSampling_doc.i.in

```diff
@@ -101,33 +101,55 @@ prob : float
 
 // ---------------------------------------------------------------------------
 
-%feature("docstring") OT::SubsetSampling::setKeepEventSample
+%feature("docstring") OT::SubsetSampling::setKeepSample
 "Sample storage accessor.
 
 Parameters
 ----------
-keepEventsample : bool
-    Whether to keep the samples that realized the event."
+keepsample : bool
+    Whether to keep the working samples at each iteration."
 
 // ---------------------------------------------------------------------------
 
-%feature("docstring") OT::SubsetSampling::getEventInputSample
+%feature("docstring") OT::SubsetSampling::getInputSample
 "Input sample accessor.
 
+Parameters
+----------
+step : int
+    Iteration index
+status : int, optional
+    Selection flag:
+
+    - 0 means the points not realizing the event
+    - 1 for points realizing the event
+    - 2 for both (default)
+
 Returns
 -------
 inputSample : :class:`~openturns.Sample`
-    Input sample that realized the event."
+    Input sample."
 
 // ---------------------------------------------------------------------------
 
-%feature("docstring") OT::SubsetSampling::getEventOutputSample
+%feature("docstring") OT::SubsetSampling::getOutputSample
 "Output sample accessor.
 
+Parameters
+----------
+step : int
+    Iteration index
+status : int, optional
+    Selection flag:
+
+    - 0 means the points not realizing the event
+    - 1 for points realizing the event
+    - 2 for both (default)
```

Comment:
How can the event simultaneously happen and not happen?

Reply:
nothing quantum here, it just return the points ok and the points failed concatenated into one sample